templates/cli/lib/commands/deploy.js.twig

```diff
@@ -907,6 +905,22 @@ const deployTeam = async ({ all, yes } = {}) => {
     }
 }
 
+const selectedOrAll = async (resourceName, questions, data) => {
+    const ids = []
+
+    const answers = await inquirer.prompt(questions);
+    ids.push(...answers[resourceName]);
+
+    if (ids.length <= 0) {
+        const all = data instanceof Function ? data() : data.map((resource) => resource.$id)
+
+        ids.push(...all);
+        tip(`Deploying All available ${resourceName}. Use the '--all' flag to execute this action intentionally\n`)
```

Comment:
Not sure I understand the message.

Shouldn't it be:
`Deploying all available ${resourceName} as none specified. Use the `--all` flag to disable this warning.`

Reply:
@gewenyu99 any suggestions?